bbs/multmail.cpp

```diff
@@ -274,11 +274,16 @@ void add_list(int *pnUserNumber, int *numu, int maxu, int allowdup) {
             bout.nl();
             bout.outstr("Already in list, not added.\r\n\n");
             i = 0;
+	    break;
           }
-          if (i) {
-            pnUserNumber[(*numu)++] = i;
-          }
-        }
+	}
+	  if (i) {
```

Comment:
I bet these two can be collapsed somehow, also isn't this already in an "if (i)" block?

Reply:
in prior section of code, "i = 0" can make this next section not fire off.